src/main/scala/io/sdkman/changelogs/GradleProfilerMigrations.scala

```diff
@@ -0,0 +1,25 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "033")
+class GradleProfilerMigrations {
+    @ChangeSet(order = "001", id = "001-add_gradle-profiler_0.12.0", author = "wolfs")
+    def migration001(implicit db: MongoDatabase): Unit = {
+      Candidate(
+        candidate = "gradle-profiler",
```

Comment:
We don't really support special characters in candidate identifiers at the moment. We usually try and keep these as short as possible. Could you try coming up with something a bit more concise?

Reply:
I'll try to figure out something. Apart from `gprofiler` or `gradleprof` nothing comes to mind.

Reply:
I'll go with `gradleprofiler`, since that is the currently known name. is that good with you?